board/imgtec/pistachio_bub/fdt.c

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (C) 2016 Imagination Technologies
+ * Author: Francois Berder <francois.berder@imgtec.com>
+ *
+ * SPDX-License-Identifier:	GPL-2.0+
+ */
+
+#include <common.h>
+
+#if defined(CONFIG_OF_LIBFDT) && defined(CONFIG_OF_BOARD_SETUP)
+
+#include <winbond-otp.h>
+#include "otp.h"
+
+DECLARE_GLOBAL_DATA_PTR;
+
+static void fixup_wifi(void *blob)
```

Comment:
should we just keep this function to fix wifi mac addresses, and write different one to overwrite calibration data ?